test/src/test_slot.py

```diff
@@ -38,7 +39,10 @@ def _Run(self):
         for arg in test.args:
             if arg.find("\"") >= 0:
                 qarg = arg.replace('"', '\\"')
-                cmd += "--lua " + "\\\"" + qarg + "\\\""
+                if platform == "darwin":
+                    cmd += "--lua " + "\\\"" + qarg + "\\\""
+                else:
+                    cmd += "--lua " + qarg
```

Comment:
should it be `platform.system()` and not just `platform` ?

Reply:
Hmm. `platform` worked for me. I tested on both OSX and Linux. Let me look at `platform.system()`.

Reply:
[sys.platform](https://docs.python.org/3/library/sys.html#sys.platform)

[sys.platform.platform()](https://docs.python.org/3/library/sys.html#sys.platform)

:-D

Reply:
and [sys.platform.system()](https://docs.python.org/3/library/platform.html#platform.system)

Reply:
I don't think that all options are identical (case of the answer)

Reply:
add `.lower()`

Reply:
After looking through the documentation, I believe what's in the code now is the correct way to go about determining the platform.